modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -329,6 +407,24 @@ indigo_core_gentable_register(
 void
 indigo_core_gentable_unregister(indigo_core_gentable_t *gentable);
 
+/*
+ * @brief resume indigo addi4()/modifyi4()/del4() when the driver gets the final status 
+ * @param op_ctx operation context 
+ * @param entry_priv driver's private data for this entry
```

Comment:
"if the async operations successfully completes, the state manager will store entry_priv for this entry"
but will this apply to both add and modify?  or just modify?

Reply:
No. Both modify and deletion the entry_priv are not used for returning value.